includes/definitions/apsoluteos.yaml

```diff
@@ -0,0 +1,14 @@
+os: apsoluteos
+text: 'ApsoluteOS'
+type: network
+icon: radware
+group: radware
+mib_dir:
```

Comment:
You should just put the mibs you need in to a directory called `mibs/radware`

Reply:
@laf 
Done